boa3/model/builtin/classmethod/tobytesmethod.py

```diff
@@ -95,6 +129,8 @@ def opcode(self) -> List[Tuple[Opcode, bytes]]:
         return []
 
     def build(self, value: Any) -> IBuiltinMethod:
+        if isinstance(value, List) and len(value) == 1:
```

Comment:
```suggestion
        if isinstance(value, Sequence) and len(value) == 1:
```
so it can value both `list` and `tuple`